modules/upgrade.adoc

```diff
@@ -19,6 +19,11 @@ You can also set a grace period for how long `PodDisruptionBudget` protected wor
 All Kubernetes objects and PVs in each {product-title} cluster are backed up as part of the {product-title} service. Application and application data backups are not a part of the {product-title} service. Ensure you have a backup policy in place for your applications and application data prior to scheduling upgrades.
 ====
 
+[NOTE]
+====
+The upgrade may take some time to start, sometimes an hour or more than that.
```

Comment:
The scheduled time was not mentioned, and according to the info mentioned in the bug, seems the upgrade takes longer time to start. 
So Shall I correct it to:- 
 "The upgrade takes longer than the scheduled time to start." 
 or 
 "When you initiate an upgrade, it can take longer than usual to begin."
 Can you please suggest which one would be more correct?

Reply:
@subhtk I am thinking there are three time here:
The time when customers create the scheduled upgrade policy
The time customers created the upgrade policy when should the policy start.
The time when the policy really start.
So I suggest, "The upgrade starting time maybe later than the scheduled time. It could be sometimes an hour or longer."

Reply:
Ack.